website/.vitepress/components/HylimoEditor.vue

```diff
@@ -97,8 +118,23 @@ onMounted(async () => {
     const editor = wrapper.getEditor()!;
     hideMainContent.value = false;
 
-    editor.getModel()?.onDidChangeContent(() => {
+    const editorModel = editor.getModel()!;
```

Comment:
How did the model change from `perhaps present` to `certainly present`?

Reply:
monaco by default creates an inmemory model, which we did here, thus this was never null.
However, I personally typically prefer ? over ! whenever possible.
As this default model could however potentially in some very rare cases interfere with our workaround, thus I decided to replace it with the safer manually created model with ddded50313b0446facc10b30f314953566d3b02e